shell/linters.sh

```diff
@@ -17,23 +17,32 @@ if [[ -n $SKIP_LINTERS ]] || [[ -n $SKIP_VALIDATE ]]; then
   exit 0
 fi
 
+# add extra, per project, linters
+linters=("$DIR/linters"/*.sh)
+if [[ -z $workspacefolder ]]; then
+    workspacefolder="$(get_repo_directory)"
+fi
```

Comment:
question: what is the case where `$workspacefolder` is not empty by this time in the script?

Reply:
This was just following existing style/conventions used in other scripts, [here](https://github.com/getoutreach/devbase/blob/1c78e07d657de9cb9c1623b76a9e26745bf4e444/shell/golangci-lint.sh#L14) and [here](https://github.com/getoutreach/devbase/blob/1c78e07d657de9cb9c1623b76a9e26745bf4e444/shell/golangci-lint.sh#L13)

I kept that convention, didn't want to break pre-existing expectation